01.1-Code_of_conduct.Rmd

```diff
@@ -0,0 +1,75 @@
+
+
+#Code of Conduct 
+(Adapted from the BioC 2020 Code of Conduct)
+Code of Conduct
+Version 1.0.2 (July 27, 2021)
+
+The Bioconductor community values an open approach to science that promotes the
+
+    sharing of ideas, code, software and expertise
+    collaboration
+    diversity and inclusivity
+    a kind and welcoming environment
+    community contributions
+
```

Comment:
I'm not sure if this does bullets.

You could add 

   - line, I think it does it ('-')